example/fr_layout.cpp

```diff
@@ -16,7 +16,6 @@
 #include <map>
 #include <vector>
 #include <boost/random/linear_congruential.hpp>
-#include <boost/progress.hpp>
```

Comment:
I know boost/progress.hpp is old stuff, but I assume program output is a lot less pretty without this?  What's the rationale for removing it?